python/cudf/cudf/tests/test_list.py

```diff
@@ -315,3 +315,23 @@ def test_contains_null_search_key(data, expect):
     expect = cudf.Series(expect, dtype="bool")
     got = sr.list.contains(cudf.Scalar(cudf.NA, sr.dtype.element_type))
     assert_eq(expect, got)
+
+
+def test_concatenate_rows_of_lists_ex1():
+    pdf = pd.DataFrame({"val": [["a", "a"], ["b"], ["c"]]})
+    gdf = cudf.from_pandas(pdf)
+
+    expect = pdf["val"] + pdf["val"]
+    got = gdf["val"] + gdf["val"]
+
+    assert_eq(expect, got)
+
+
+def test_concatenate_rows_of_lists_ex2():
```

Comment:
I think we can just combine these into one test and parametrize over the data

Reply:
I separated the tests because the (expect,got) values are from indexing the pandas and cudf dataframes' columns. 
Since I am testing on dataframes with different column labels, I don't think parameterize  over` data` will be helpful
in this case?

Reply:
I separated the tests because the (`expect,got`) values are from indexing the `pandas` and `cudf `dataframes' `columns.` 

And since I am testing on dataframes with different column labels,  parameterize  over` data` alone won't be helpful....

unless  I parameterize over `pdf, gdf, expect, got`. Is that acceptable?

Reply:
All good this works fine 👍